docs/reference/search/rank-eval.asciidoc

```diff
@@ -346,16 +346,16 @@ http://olivier.chapelle.cc/pub/err.pdf[Expected reciprocal rank for graded relev
 It is based on the assumption of a cascade model of search, in which a user 
 scans through ranked search results in order and stops at the first document 
 that satisfies the information need. For this reason, it is a good metric for 
-question answering and navigation queries, but less so for survey oriented 
+question answering and navigation queries, but less so for survey-oriented 
 information needs where the user is interested in finding many relevant 
 documents in the top k results.
 
 The metric models the expectation of the reciprocal of the position at which a 
-user stops reading through the result list. This means that relevant document in 
-top ranking positions will contribute much to the overall score. However, the 
-same document will contribute much less to the score if it appears in a lower 
-rank, even more so if there are some relevant (but maybe less relevant) 
-documents preceding it. In this way, the ERR metric discounts documents which 
+user stops reading through the result list. This means that a relevant documents 
```

Comment:
```suggestion
user stops reading through the result list. This means that a relevant document
```
I think this should be singular now after your changes.

Reply:
nice catch, thanks!
I'm glad it's helpful--I love copy editing :)